zowe-cli-id-federation-plugin/src/api/templates/job.jcl

```diff
@@ -0,0 +1,24 @@
+//IDF{{esm}} JOB ({{account}}),CLASS=A,MSGCLASS=X{{sysaff}}
```

Comment:
Make it sense to put ESM as part of the Job name? There cannot be run multiple different ESM on the same system, right?

Reply:
I'm not sure why it's here, but I guess it could be a simple way to know there was a mistake

Reply:
It make sense to me. The Jobname is then little bit more unique because ESM parameter is mandatory, while system param is optional (it is used only for SYSAFF which does not need to be included).